tests/test_show_version.py

```diff
@@ -0,0 +1,127 @@
+import pathlib
+
+import pytest
+
+from komodo.show_version import (
+    get_version,
+    guess_komodo_path,
+    guess_komodoenv_path,
+    guess_release,
+    parse_args,
+    read_config,
+)
+
+
+def test_read_config_file(mock_config_file):
+    """Test the function that reads sectionless INI files."""
+    config = read_config("foo.cfg")
+    mock_config_file.assert_called_once_with("foo.cfg", "rt", encoding="utf-8")
+    assert config["home"] == "/foo/bar/komodo-release-0.0.1/root/bin"
+
+
+def test_guess_release(mock_komodo_env_vars):
+    """Simply reads an environment variable."""
+    assert guess_release() == "komodo-release-0.0.1"
+
+
+def test_guess_release_fails_without_env_var():
+    """
+    Check that the script exists gracefully if not run in a komodo environment.
+    """
+    with pytest.raises(SystemExit) as excinfo:
```

Comment:
`excinfo` is very difficult to pronounce. What do you think about refactoring this name?

Reply:
It's the name they use in the `pytest` docs, but I'm happy to change it. `exception_info`?

Reply:
Done

Reply:
That I did not know, thanks for considering.